src/FNA3D_Driver_Vulkan.c

```diff
@@ -3160,9 +3129,7 @@ static uint8_t VULKAN_INTERNAL_FindAvailableTextureMemory(
 
 	if (cpuAllocation)
 	{
-		memoryPropertyFlags =
-			VK_MEMORY_PROPERTY_HOST_VISIBLE_BIT |
-			VK_MEMORY_PROPERTY_HOST_COHERENT_BIT;
+		memoryPropertyFlags = VK_MEMORY_PROPERTY_HOST_COHERENT_BIT;
```

Comment:
This should be dropped as well, currently you won't find the memory host type on nvidia because its flags is `0`.

Reply:
Is it legal to allocate a texture on non-coherent memory?

Reply:
Well I just tested it and it works, so that answers that question!